examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -163,7 +164,9 @@ def save_model_card(
         "template:sd-lora",
     ]
     if "playground" in base_model:
-        tags.extend(["playground", "playground-diffusers"])
+        tags.extend(["playground", "playground-diffusers", "edm-training"])
+    elif edm_training:
+        tags.extend(["stable-diffusion-xl", "stable-diffusion-xl-diffusers", "edm-training"])
```

Comment:
We should also configure the `scheduler_type` in the inference params here otherwise the API results will be funky.